addon/authenticators/oauth2-password-grant.js

```diff
@@ -214,6 +214,36 @@ export default BaseAuthenticator.extend({
     method also schedules refresh requests for the access token before it
     expires.__
 
+    The server responses to are expected to look as defined in the spec (see
```

Comment:
Woops, just missed that one: I think it would be

```
The server responses are expected to look [...]
```

Reply:
fixed 👍